backend/submissions/models.py

```
@@ -0,0 +1,142 @@
+            response = self.table.update(data).eq('id', submission_id).execute()
+            return response.data[0] if response.data else None
+        except Exception as e:
+            print(f"Error updating submission: {str(e)}")
+            return None
+
+    def delete(self, submission_id):
+        """
+        Delete a submission
+        """
+        try:
+            response = self.table.delete().eq('id', submission_id).execute()
```

Ruff (0.11.9)

Local variable `response` is assigned to but never used

Remove assignment to unused variable `response`

(F841)